test/e2e/fleet_test.go

```diff
@@ -945,6 +945,36 @@ func TestFleetWithZeroReplicas(t *testing.T) {
 	assert.Empty(t, list)
 }
 
+// TestFleetWithLongLabels ensures that we can not create a fleet
+// with label over 64 chars
+func TestFleetWithLongLabels(t *testing.T) {
+	t.Parallel()
+	client := framework.AgonesClient.AgonesV1()
+	fleetSize := int32(1)
+	flt := defaultFleet(defaultNs)
+	flt.Spec.Replicas = fleetSize
+	flt.Spec.Template.ObjectMeta.Labels = make(map[string]string)
+	nameLen := validation.LabelValueMaxLength + 1
+	key := make([]byte, nameLen)
+	for i := 0; i < nameLen; i++ {
+		key[i] = 'f'
+	}
+	flt.Spec.Template.ObjectMeta.Labels["label"] = string(key)
```

Comment:
Should be able to just do

`flt.Spec.Template.ObjectMeta.Labels["label"] = strings.Repeat("f", validation.LabelValueMaxLength + 1)`

See https://golang.org/pkg/strings/#Repeat

Reply:
Thanks, applying this to all connected code parts